spectec/src/frontend/elab.ml

```diff
@@ -241,15 +241,15 @@ let rec equiv_typ env t1 t2 =
     (Print.string_of_typ t1) (Print.string_of_typ t2)
     (t1.it = t2.it);
   *)
-  t1.it = t2.it ||
-  match expand env t1, expand env t2 with
+  t1.it = t2.it
+  ||
```

Comment:
What?

Reply:
I think this is because of a match, and couldn't find a way to change it.